src/gui/src/CapriceOptions.cpp

```diff
@@ -247,15 +249,22 @@ CapriceOptions::CapriceOptions(const CRect& WindowRect, CWindow* pParent, CFontE
     m_pDropDownCPCLanguage->SelectItem(CPC.keyboard);
     m_pDropDownCPCLanguage->SetIsFocusable(true);
     // option 'kbd_layout' which is the platform keyboard layout (i.e. the PC keyboard layout)
-    m_pLabelPCLanguage    = new CLabel(CPoint(10,33), m_pGroupBoxTabInput, "PC Keyboard language");;
+    m_pLabelPCLanguage    = new CLabel(CPoint(10,33), m_pGroupBoxTabInput, "PC Keyboard layout");;
     m_pDropDownPCLanguage = new CDropDown(CRect(CPoint(130,31),140,16), m_pGroupBoxTabInput, false, 14);
-    m_pDropDownPCLanguage->AddItem(SListItem("English"));
-    m_pDropDownPCLanguage->AddItem(SListItem("French"));
-    m_pDropDownPCLanguage->AddItem(SListItem("Spanish"));
-    m_pDropDownPCLanguage->SetListboxHeight(3);
-    m_pDropDownPCLanguage->SelectItem(CPC.kbd_layout);
-    m_pDropDownPCLanguage->SetIsFocusable(true);
 
+    mapFileList = listDirectoryExt(CPC.resources_path, "map");
+	unsigned int currentMapIndex = 0;
```

Comment:
Right. I fixed this to use the old style (no tab). BTW this code has a small problem: if the layout file is wrong/empt/undefined, the first layout file found will be selected by default, which is a bit misleading. I do not really know how to handle this.